cookiecutter.json

```
@@ -5,6 +5,12 @@
     "author_name": "",
     "author_email": "",
     "project_url": "",
+    "sourcecode_url": "",
+    "github_repo": "{{ cookiecutter.sourcecode_url | replace('https://github.com/', '') }}",
```

I wonder if we could do this:
```suggestion
    "github_repo": "{{ cookiecutter.sourcecode_url | replace('https://github.com/', '') if 'https://github.com' in cookiecutter.sourcecode_url else '' }}",
```